modules/snooker/src/main/ts/ephox/snooker/model/RunOperation.ts

```diff
@@ -104,8 +119,10 @@ type Adjustment = <T extends Structs.DetailNew>(table: Element, grid: Structs.Ro
 type PostAction = (e: Element) => void;
 type GenWrap<GW extends GeneratorsWrapper> = (g: Generators) => GW;
 
+export type OperationCallback<T> = (wire: ResizeWire, table: Element<HTMLTableElement>, target: T, generators: Generators, direction: BarPositions<ColInfo>) => Option<RunOperationOutput>;
```

Comment:
Should the generic here be `T extends CombinedTargets`?

Reply:
there's other types it can be e.g. TargetPaste so I don't think it should be.